presto-cache/src/main/java/com/facebook/presto/cache/filemerge/FileMergeCacheManager.java

```diff
@@ -13,11 +13,13 @@
  */
 package com.facebook.presto.cache.filemerge;
 
```

Comment:
Remove the first TODO in this class: `TODO: Make cache eviction based on cache...`